gradle.properties

```diff
@@ -28,12 +28,12 @@ jee7api=https://docs.oracle.com/javaee/7/api
 jeeapi=https://docs.oracle.com/javaee/6/api
 jakartaapi=https://jakarta.ee/specifications/platform/9/apidocs
 kafkaapi=https://kafka.apache.org/11/javadoc
-micronautdiscoveryapi=https://micronaut-projects.github.io/micronaut-discovery-client/latest/api/io/micronaut/discovery
-micronauthystrixapi=https://micronaut-projects.github.io/micronaut-netflix/latest/api/io/micronaut/configuration/hystrix
-micronautribbonapi=https://micronaut-projects.github.io/micronaut-netflix/latest/api/io/micronaut/configuration/ribbon
-micronautcacheapi=https://micronaut-projects.github.io/micronaut-cache/latest/api/io/micronaut/cache
-micronautreactorapi=https://micronaut-projects.github.io/micronaut-reactor/latest/api/io/micronaut/reactor
-micronautspringapi=https://micronaut-projects.github.io/micronaut-spring/latest/api/io/micronaut/spring
+micronautdiscoveryapi=https://micronaut-projects.github.io/micronaut-discovery-client/latest/api
+micronauthystrixapi=https://micronaut-projects.github.io/micronaut-netflix/latest/api
+micronautribbonapi=https://micronaut-projects.github.io/micronaut-netflix/latest/api
+micronautcacheapi=https://micronaut-projects.github.io/micronaut-cache/latest/api
+micronautreactorapi=https://micronaut-projects.github.io/micronaut-reactor/latest/api
+micronautspringapi=https://micronaut-projects.github.io/micronaut-spring/latest/api
```

Comment:
This would break the existing docs, wouldn't it?

Reply:
That's actually the other way around: I was puzzled that I got error messages when I built locally, that the links were not valid. I then realized that they needed to end at `api` otherwise javadoc won't find the package list.